src/flaresolverr_service.py

```diff
@@ -264,12 +265,19 @@ def _evil_logic(req: V1RequestBase, driver: WebDriver, method: str) -> Challenge
 
     challenge_res = ChallengeResolutionResultT({})
     challenge_res.url = driver.current_url
-    challenge_res.status = 200  # todo: fix, selenium not provides this info
+    try:
+        challenge_res.status = driver.requests[
+            0].headers  # Todo this is the status code BEFORE solving the challenge, so most likely a 403
+    except:
+        challenge_res.status = 200
     challenge_res.cookies = driver.get_cookies()
     challenge_res.userAgent = utils.get_user_agent(driver)
 
     if not req.returnOnlyCookies:
-        challenge_res.headers = {}  # todo: fix, selenium not provides this info
+        try:
+            challenge_res.headers = driver.requests[0].headers
```

Comment:
setting `-1` instead of `0` should theoretically get the status code AFTER challenge completion